packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -644,6 +645,7 @@ export default NavBar;
 
 export const LoadingNavBar = () => {
   const ref = React.useRef<HTMLDivElement>(null);
+  useInitialForcedMenuState();
```

Comment:
I decided to have a separate hook to use here, as otherwise the "big" state manager hook sets event listeners and other things that are not necessary for the "loading" navbar container.